hs/src/Reach/Eval/Core.hs

```diff
@@ -2965,13 +2985,8 @@ evalPrim p sargs =
       foldable_includes <- lookStdlib "Foldable_includes"
       case tup of
         SLV_Tuple _ vs -> do
-          let typeMatch v = do
-                mTA <- typeOfM v
-                case mTA of
-                  (Just (t, _a)) -> return $ t == nt
-                  _ -> return $ False
-          vsMatched <- filterM typeMatch vs
-          evalApplyVals' foldable_includes [(lvl, (SLV_Array at nt vsMatched)), (lvl, needle)]
+          v_ret <- evalOrMap (\v -> snd <$> evalPolyEq Public v needle) vs
```

Comment:
I want this to be the only case and get `vs` from `explodeTupleLike`

Reply:
IE you want to remove the working dynamic case and make this a static-only feature?

Reply:
`explodeTupleLike` works exactly like your DLVar case: https://github.com/reach-sh/reach-lang/blob/82d3f7c58a444ebcf9f7a4be50fe8b41f38ffa6f/hs/src/Reach/Eval/Core.hs#L2441-L2442

Reply:
Oh, somehow I missed that.